### DISCUSSION TITLE
Offline mode

### DISCUSSION THREAD

**Discussion by NedPool** [DonutWare/Fladder#187] [Ideas]
### Summary

First, thank you for the excellent client!
There is already the ability to sync files for local playback, but offline support is desperately needed.

Very frustrating when I know I have the files on my device, but I can't play anything because it won't load the item/season page.

### Basic Example

When offline, have the ability to play locally synced media from the Sync page or elsewhere.

### Drawbacks

Could increase local cache requirements.

**Comment by NedPool** [comment]
I take that back... on further testing, it isn't actually USING the downloaded file from the sync for playback; it is still streaming from the server.

Please please please enable local file syncing with true offline playback!!

**Comment by PartyDonut** [comment]
While I agree the UI itself isn't very clear, it does work exactly like that.
Currently there are 2 ways a synced file gets played.

1. Airplane mode, when no internet connection is available and it's synced the offline file is played.
2. Long press the play button on the details page and under Playback type press the offline option (only available if synced)

**Reply by NedPool** [reply]
Gotcha, ok, I'll keep trying but I haven't been able to get any offline playback working. My client (android and windows) also streamed a file when it was synced locally (verified the file exists in the windows save location).

**Reply by NedPool** [reply]
@PartyDonut 
Loading the details page is the issue now. It simply does not load. Most likely to not having the correct cached assets, which I can't tell it to sync.

Attempting to get to the details page for any content (in order to get to a play button), fails.

**Reply by PartyDonut** [reply]
@NedPool  I've created a ticket for epsiode details not loading #189. Other items seem to load the details just fine from what I can tell.
